### DISCUSSION TITLE
Upgrade 0.10.0 to latest for TYPO3 v11

### DISCUSSION THREAD

**Discussion by WSBusch** [oliverklee/ext-seminars#2597] [Q&A] [answered]
Hello,

we may update the extension "seminars" in a TYPO3 version 6 to version 11 for a client.
Does anyone have a best practice to upgrade from version 0.10.0 to the current version for TYPO3 v11?
Are there any special steps to follow? When do upgrade wizards come into play, if applicable?

Thank you very much for your answers :-)

**Comment by oliverklee** [comment, marked as answer]
You'll need to have at least each major version (1.x, 2.x, etc.) of seminars installed as the upgrade wizards (and, in ye olde days, the EM upgrade scripts) only update the data from one major version to the next.

You'll also need to sort out the dependency versions: seminars, oelib, feuserextrafields, mkforms (for older seminars versions), TYPO3 so that you have a working installation of at least each seminars major version.

**Reply by oliverklee** [reply]
(For a customer project of mine where I needed to upgrade from TYPO3 8LTS to 11LTS, I created a spreadsheet to plan in which order I would raise TYPO3, oelib, onetimeaccount, feuserextrafields and seminars to which version.)
